static/html/js/clientChecker.js

```diff
@@ -15,7 +15,7 @@ this.ClientChecker = function(callback, checkInterval) {
 	var checkClientStatus = function() {
 		if(!clientIsAvailable) {
 			$.each(that.clientUrls, function(index, url) {
-				$.get(url + statusParam)
+				$.get(url.url + statusParam)
```

Comment:
Name somewhat misleading. How about `that.eidClients` instead of `that.clientUrls` and `$.each(that.eidClients, function(index, eidClient))`?